src/pages/GetInvolvedPage.tsx

```diff
@@ -1,5 +1,36 @@
+import { Map, MapGeoJsonSource, MapContainer } from "@/atoms/MapAtoms";
+import { Theme } from "@/theme/Theme";
+import  BPDA from "@/data/BPDA";
+import { Marker } from "react-map-gl";
+
+const BPDADevelopmentLayer = () => (
+  <MapGeoJsonSource
+    data={BPDA}
+    id="BPDA"
+    type="circle"
+    color={Theme.colors.blue}
+  />
+);
+
 export default function GetInvolvedPage() {
   return (
-    <h1>Get Involved</h1>
-  )
+    <MapContainer>
+      <Map mapStyle={Theme.map.dark}>
+        <>
+          <BPDADevelopmentLayer />
+          {BPDA.map((data:any) => (
```

Comment:
I assume that you want to refer to `BPDA.features`, rather than `BPDA`, when iterating through coordinate pairs?
```suggestion
          {BPDA.features.map((data:any) => (
```

Reply:
Yes I agree with above as well, `BPDA.features` seems like the array to iterate over to grab the coordinates